network/snat/snat_linux.go

```diff
@@ -474,3 +488,20 @@ func (client *Client) addVlanDropRule() error {
 	_, err = client.plClient.ExecuteCommand(vlanDropAddRule)
 	return err
 }
+
+// This function enables ip forwarding in VM and allow forwarding packets from the interface
+func (client *Client) EnableIPForwarding() error {
+	// Enable ip forwading on linux vm.
+	// sysctl -w net.ipv4.ip_forward=1
+	_, err := client.plClient.ExecuteCommand(enableIPForwardCmd)
+	if err != nil {
+		return errors.Wrap(err, "enable ipforwarding command failed")
+	}
```

Comment:
hm, I wish we didn't have this in the codebase already... I generally don't think that it's our responsibility to configure the host like this.